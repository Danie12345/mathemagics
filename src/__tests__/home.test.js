import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';

const Home = require('../components/home/Home').default;

test('loads the home with everything', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
