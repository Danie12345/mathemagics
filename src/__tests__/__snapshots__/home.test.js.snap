// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads the container with everything 1`] = `
<div
  className="section homeContainer"
>
  <h2
    className="title"
  >
    Welcome to the Mathemagics website!
  </h2>
  <p
    className="introduction"
  >
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Aliquam, in! Officia ex, corrupti error nemo doloribus labore ratione repudiandae laboriosam repellat neque asperiores, sint eos assumenda illum dicta voluptate. Dolorem temporibus veritatis facere explicabo doloremque magnam iure, repellat facilis hic necessitatibus odit pariatur voluptatum vero, veniam saepe deserunt aliquam dignissimos nisi minima perspiciatis officiis maiores nobis labore provident.
  </p>
  <p
    className="introduction"
  >
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Aliquam, in! Officia ex, corrupti error nemo doloribus labore ratione repudiandae laboriosam repellat neque asperiores, sint eos assumenda illum dicta voluptate. Dolorem temporibus veritatis facere explicabo doloremque magnam iure, repellat facilis hic necessitatibus odit pariatur voluptatum vero, veniam saepe deserunt aliquam dignissimos nisi minima perspiciatis officiis maiores nobis labore provident.
  </p>
</div>
`;

exports[`loads the home with everything 1`] = `
<div
  className="section homeContainer"
>
  <h2
    className="title"
  >
    Welcome to the Mathemagics website!
  </h2>
  <p
    className="introduction"
  >
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Aliquam, in! Officia ex, corrupti error nemo doloribus labore ratione repudiandae laboriosam repellat neque asperiores, sint eos assumenda illum dicta voluptate. Dolorem temporibus veritatis facere explicabo doloremque magnam iure, repellat facilis hic necessitatibus odit pariatur voluptatum vero, veniam saepe deserunt aliquam dignissimos nisi minima perspiciatis officiis maiores nobis labore provident.
  </p>
  <p
    className="introduction"
  >
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Aliquam, in! Officia ex, corrupti error nemo doloribus labore ratione repudiandae laboriosam repellat neque asperiores, sint eos assumenda illum dicta voluptate. Dolorem temporibus veritatis facere explicabo doloremque magnam iure, repellat facilis hic necessitatibus odit pariatur voluptatum vero, veniam saepe deserunt aliquam dignissimos nisi minima perspiciatis officiis maiores nobis labore provident.
  </p>
</div>
`;
